scripts/data/cleanWDPA.yml

```diff
@@ -1,3 +1,4 @@
+# For review PR
```

Comment:
For the YML files and the JSON file: adding a lifecycle tag with value "reviewed" should be the way to flag them. That way, once merged, they will be tagged appropriately.